bin/pip-compile

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -e
+
+# Click requires us to ensure we have a well configured environment to run
+# our click commands. So we'll set our environment to ensure our locale is
+# correct.
+export LC_ALL="${ENCODING:-en_US.UTF-8}"
+export LANG="${ENCODING:-en_US.UTF-8}"
+
+COMMAND_ARGS=$@
```

Comment:
Here's, it might actually make a difference to use `"$@"` with double quotes.

From `man bash`:
```
       @      Expands to the positional parameters, starting from one.   When  the  expansion  occurs  within  double
              quotes,  each  parameter  expands to a separate word.  That is, "$@" is equivalent to "$1" "$2" ...  If
              the double-quoted expansion occurs within a word, the expansion of the first parameter is  joined  with
              the  beginning  part  of  the original word, and the expansion of the last parameter is joined with the
              last part of the original word.
```

So the difference is if one of the parameters is `"a b"`, with `"@"`, it stays that way, and with `$@` it becomes 2 distinct parameters.

Reply:
Updated to use quotes everywhere we do this.